tilequeue/wof.py

```diff
@@ -327,13 +327,24 @@ def failure(reason):
     edtf_cessation = _normalize_edtf(props.get('edtf:cessation'))
     edtf_deprecated = _normalize_edtf(props.get('edtf:deprecated'))
 
+    # check that the dates are valid first to return back a better error
+    inception_earliest = edtf_inception.date_earliest()
+    cessation_latest = edtf_cessation.date_latest()
+    deprecated_latest = edtf_deprecated.date_latest()
+    if inception_earliest is None:
+        return failure('invalid edtf:inception: %s' % props['edtf:inception'])
```

Comment:
It's possible for `props` not to have an `edtf:...` key, which is hidden by `_normalize_edtf` returning a valid `EDTF` object when given `None`. This might cause a `KeyError` here, though?

Perhaps `failure('invalid edtf:inception: %r' % props.get('edtf:inception'))`?